### PR TITLE
Add front group role

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2173
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/215
https://github.com/open-orchestra/open-orchestra-model-interface/pull/298
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/754
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/255

### CODE REVIEW

File: DisplayBundle/DisplayBlock/Strategies/AbstractMenuStrategy.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+namespace OpenOrchestra\DisplayBundle\DisplayBlock\Strategies;
+
+use OpenOrchestra\FrontBundle\Security\ContributionActionInterface;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+
+/**
+ * Class AbstractMenuStrategy
+ */
+abstract class AbstractMenuStrategy extends AbstractDisplayBlockStrategy
```
Comment (alavieille) [inline DisplayBundle/DisplayBlock/Strategies/AbstractMenuStrategy.php]: use it also in footerStrategy

**Comment by itkg-nanne** [comment]
update done @alavieille

File: DisplayBundle/DisplayBlock/Strategies/AbstractNodeRepositoryStrategy.php [hunk 2]
```diff
@@ -3,12 +3,13 @@
 namespace OpenOrchestra\DisplayBundle\DisplayBlock\Strategies;
 
 use OpenOrchestra\FrontBundle\Security\ContributionActionInterface;
+use OpenOrchestra\ModelInterface\Model\ReadNodeInterface;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 
 /**
- * Class AbstractMenuStrategy
+ * Class AbstractNodeRepositoryStrategy
```
Comment (alavieille) [inline DisplayBundle/DisplayBlock/Strategies/AbstractNodeRepositoryStrategy.php]: why repository in the naming ?

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by itkg-nanne** [comment]
update done @alavieille

**Pull request merged**